infrastructure/quick-deploy/aws/all/versions.tf

```diff
@@ -0,0 +1,36 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.18.0"
```

Comment:
shouldn't this be updated to `>=` like in the previous files

Reply:
The recommendation is to use `>=` in modules, an `~>` in the root (if required). I don't think `~>` is required here, but I wanted to be conservative.

Reply:
https://developer.hashicorp.com/terraform/language/providers/requirements#best-practices-for-provider-versions